sched/Kconfig

```diff
@@ -13,14 +13,6 @@ menuconfig DISABLE_OS_API
 		disable functions unless you want to restrict usage
 		of those APIs.
 
-		There are certain dependency relationships in these
-		features.
-
-		1) mq_notify logic depends on signals to awaken tasks
-		waiting for queues to become full or empty.
-		2) pthread_condtimedwait() depends on signals to wake
-		up waiting tasks.
-
```

Comment:
Everything else looks good, but why remove this section? Is it stale?

Reply:
That section was meaningful only when we had CONFIG_DISABLE_SIGNALS. Makes little sense to list dependencies to features that cannot in fact be removed. Wording "in these features" refers to the set of features than can be disabled by subitems of CONFIG_DISABLE_OS_API so it is misleading to imply that signals are part of it.